### PR TITLE
Create krushkals.c

### PR DESCRIPTION
**Pull request by ankit-akash**
#### Description of Change
-C Programming Implementation of Kruskal’s Algorithm

#### References

https://www.geeksforgeeks.org/kruskals-minimum-spanning-tree-algorithm-greedy-algo-2/
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes: 
Kruskal’s algorithm is a greedy algorithm in [graph theory](https://www.britannica.com/topic/graph-theory) that finds a minimum spanning tree for a connected weighted graph. It finds a subset of the edges that forms a tree that includes every vertex, where the total weight of all the edges in the tree is minimized.

### CODE REVIEW

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Hey there! Thank you for submitting a PR! 😄 Sadly, the documentation is not up to our [**standards**](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md). Please update the file accordingly. **Let us know if you need any help here or on our [Discord server](https://the-algorithms.com/discord).** 😃

**Comment by ankit-akash** [comment]
do you want me to add comments in the program?

**Comment by Panquesito7** [comment]
> do you want me to add comments in the program?
> 
> #hactoberest_accepted

That would be great, thanks! You can take a look at our [typical structure of a program](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) if you're confused on how to do that. 🙂

File: greedy_approach/krushkals.c [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+ * Kruskal’s algorithm is a greedy algorithm in graph theory that finds a minimum spanning tree for a connected weighted graph.
+ * theory that finds a minimum spanning tree for a connected weighted graph.
+ * It finds a subset of the edges that forms a tree that includes every vertex, * where the total weight of all the edges in the tree is minimized.
+ * @author [Ankit Akash](https://github.com/ankit-akash)
+ */
+ 
+  #include <stdio.h>
+    #include <conio.h>
+    #include <stdlib.h>
+
```
Comment (Panquesito7) [inline greedy_approach/krushkals.c:14]: Please add a one-line description of what the library/header is for (**see the example below**).

```c
#include <stdio.h>     /// for IO operations
#include <assert,h>    /// for assert
```

File: greedy_approach/krushkals.c [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    int i, j, k, a, b, u, v, n, ne = 1;
```
Comment (Panquesito7) [inline greedy_approach/krushkals.c:15]: Do not use public variables. Use them in functions, classes, or structures.

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
Please check our [**typical structure of a program**](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#typical-structure-of-a-program) to make your file up to our standards.
Let us know if you need any help with that. Thank you. 🙂

**Pull request closed without merging**